src/objects/page/page.scss

```diff
@@ -27,8 +27,25 @@
 
 .o-page__content {
   grid-row: 2;
+  position: relative;
 }
 
 .o-page__footer {
   grid-row: 3;
 }
+
+/**
+ * An optional `alert` block can be used to pass an Alert component. The following
+ * styles position the Alert to match design intent.
+ */
+
+.o-page__content-alert {
+  display: flex;
+  inline-size: 100%;
+  inset-block-start: -1.5em;
+  inset-inline-start: 50%;
+  justify-content: center;
+  position: absolute;
+  transform: translateX(-50%);
```

Comment:
I think if this content were in a zero-height row you might be able to simplify a lot of the positioning styles?

Here's an example: https://codepen.io/tylersticka/pen/XWEddmx/54a5a93880b50b3ceb5b0df91ec256b9??editors=1100

Reply:
Thanks for the suggestion, @tylersticka!

https://github.com/cloudfour/cloudfour.com-patterns/pull/1921/commits/c45d0678576c319ac8db62b8b018770029094f19